espnet/nets/pytorch_backend/transducer/causal_conv1d.py

```diff
@@ -0,0 +1,59 @@
+"""CausalConv1d module definition for transformer decoder."""
+
+import torch
+
+
+class CausalConv1d(torch.nn.Module):
+    """CausalConv1d module for transformer decoder.
+
+    Args:
+        idim (int): dimension of inputs
+        odim (int): dimension of outputs
+        kernel_size (int): size of convolving kernel
+        stride (int): stride of the convolution
+        dilation (int): spacing between the kernel points
+        groups (int): number of blocked connections from ichannels to ochannels
+        bias (bool): whether to add a learnable bias to the output
+
+    """
+
+    def __init__(
+        self, idim, odim, kernel_size, stride=1, dilation=1, groups=1, bias=True
+    ):
+        """Construct a CausalConv1d object."""
+        super(CausalConv1d, self).__init__()
```

Comment:
This is just a comment.
Please avoid python2 syntax in espnet2.
e.g.

```
# bad
        super(CausalConv1d, self).__init__()
# good
        super().__init__()
```

You must write type annotations instead of type information in docstring in epsnet2. This is mandatory.

Reply:
OK! However, as far as I know, this syntax is used almost everywhere in espnet1? Should I change that for all classes related to this PR?

Reply:
No no, this is just a comment about espnet2. Sorry for confusing.

Ideally saying, espnet1 should be refactored everywhere, not only about this point, but I gave up to refactor espnet1 once because espnet1 has too much problems. This is main reason why I decided to create espnet2.